src/components/Visualization.css

```diff
@@ -22,6 +22,13 @@
     height: calc(100vh - 77px);
 }
 
+.stepHandle {
+    width: 8px !important;
```

Comment:
@lordrip I tried to find a way around having to use `!important` here but couldn't figure it out. If you have any ideas though let me know! :)

Reply:
mmm I see :thinking:, I think that it's similar to what @igarashitm had the other day, in relation to the style specificity.

One way to remove the `!important` bit could be the following:
```css
.stepNode .stepHandle.react-flow__handle {
    width: 8px;
    height: 8px;
    background: var(--pf-global--BorderColor--200);
    border-radius: 100%;
}
```
it's the same style as before but now I tried to provide more specificity, if I'm not mistaken, we're saying now:

> _All children from nodes containing the `.stepNode` class AND that also have the `stepHandle` and `react-flow__handle` classes together_

That being said, I was curious why the React flow class won instead of ours, and it seems that it's related to the definition order, probably the react flow styles are added below ours, hence getting the winning hand for the colliding specificity, but this is just a theory :sweat_smile: 

I found [this link](https://www.w3.org/TR/CSS2/cascade.html#cascading-order) from an older draft very interesting in regards to what gets applied when. :smiley: